### PR TITLE
Final_notebook

### PR DESCRIPTION
**Pull request by svjai**
This PR contains the changes appended on top of the last day progress made

1. Completed Notebook
2. model comparison and hyperparameter tuning 
3. feature importance and feature selection
4. submission repo is created

- [x]  Fine tuning and feature importance (coefficients of the features for every model)
- [x] submission repo created and HTML,PDF files are added 
- [x] metrics and evaluation of the models used 
- [x] Lasso (L1 regularied model) is performing good with unseen data --> mse score of 0.04
- [x] Random forest has    overfitting issues and the ridge regression, parameter tuned gradient boosting models are performing decently w.r.t R2 scpore
- [ ] markdown and few left-out comments 
- [ ] completion of the report (10 pages) and PPT (20 slides) on-progress {ETA: this weekend}

### CODE REVIEW

**Review by NILodio — APPROVED** [review]
Really good job!

**Comment by NILodio** [comment]
Really goog Job!!! @svjai
